python/dlisio/plumbing/frame.py

```diff
@@ -396,19 +391,8 @@ def curves(self):
         DataFrame
 
         >>> curves = frame.curves()
-        >>> if frame.index_type:
-        >>>     indexname  = curves.dtype.names[0]
-        >>>     curvenames = curves.dtype.names[1:]
-        >>>     pdcurves = pd.DataFrame(curves[list(curvenames)], index=curves[indexname])
-        >>>     pdcurves.index.name = indexname
-        >>> else:
-        >>>     pdcurves = pd.DataFrame(curves)
-
-        Let's walk through this. Firstly, check that *index_type* is not None,
-        if it is, there is no index channel to set. Secondly, note how the
-        curvenames are all stored in the array itself. Then all that is needed
-        is to extract the first name as index and let the rest be added to the
-        DataFrame as normal Channels.
+        >>> pdcurves = pd.DataFrame(curves.dtype.names[1:], index=curves[f.index])
```

Comment:
Shouldn't it be
`curves[list(curves.dtype.names[1:])] `
then? I am getting lost 

Maybe repeated index is easier for the user?

Reply:
I'd go for the repeated index for simplicity
(And yes, it should have been a list)